doc/whats_new.rst

```diff
@@ -37,6 +37,11 @@ Fixes
 -----
 
 - Example plot_seed_to_voxel_correlation now really saves z-transformed maps.
+- Fix bug in region_extractor.connected_regions which did not systematically
+use the mask_img when provided
+- Fix bug in regions.RegionExtractor which did not use the mask_img when
+provided
+
```

Comment:
Keeping it consistent with the grammar of other entries, rewrite this as
`region_extractor.connected_regions`  & `regions.RegionExtractor` nowcorrectly use the provided mask_img.

The first letter of each line(not sentence) should be aligned in RestructuredText. In the text in your commit,
the letter 'u' of use should be right under the letter 'F' of Fix.

Reply:
Let CircleCi complete first, so the next push done the same day will be faster.